_test/test_admin/test_pc_spec_config.m

```diff
@@ -85,8 +121,21 @@ function test_set_pc_type(obj)
             cm.this_pc_type = 'a_mac';
             assertEqual(cm.this_pc_type,'a_mac');
             assertEqual(cm.pc_config_num,3);
+            
+            kpc = cm.known_pc_types;
+            nconf = numel(kpc);
+            for i=1:nconf
+                cm.this_pc_type = i;
+                assertEqual(cm.this_pc_type,kpc{i});
+                assertEqual(cm.pc_config_num,i);
+            end
+            for i=1:nconf
```

Comment:
what is the purpose of the second for-loop here?

Reply:
you can set up configuration both by number and by name

Reply:
These two test cases should be split into two test functions then. 

Looking at the full function there are five distinct test cases be checked so these should be split out:
- throws error if unknown pc type
- throws error in invalid index  (which needs to check -1 and nconf+1 edges)
- sets 'amac' --> why is this in the sets PC case when it's not part of the know_pc_types array?
- sets pc by array index
- sets pc by name